src/adapter/src/coord/sequencer/inner/create_continual_task.rs

```diff
@@ -7,15 +7,34 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+use std::sync::Arc;
+
+use mz_catalog::memory::objects::{
+    CatalogEntry, CatalogItem, MaterializedView, Table, TableDataSource,
+};
+use mz_compute_types::sinks::{
+    ComputeSinkConnection, ContinualTaskConnection, PersistSinkConnection,
+};
+use mz_expr::visit::Visit;
+use mz_expr::{Id, LocalId};
 use mz_ore::instrument;
+use mz_repr::adt::mz_acl_item::PrivilegeMap;
+use mz_repr::optimize::OverrideFrom;
 use mz_sql::names::ResolvedIds;
-use mz_sql::plan;
+use mz_sql::plan::{self, HirRelationExpr};
+use mz_sql::session::metadata::SessionMetadata;
+use mz_storage_client::controller::{CollectionDescription, DataSource, DataSourceOther};
 
+use crate::catalog;
 use crate::command::ExecuteResponse;
 use crate::coord::Coordinator;
 use crate::error::AdapterError;
+use crate::optimize::dataflows::dataflow_import_id_bundle;
+use crate::optimize::{self, Optimize};
 use crate::session::Session;
+use crate::util::ResultExt;
 
+// TODO(ct): Big oof. Dedup a bunch of this with MVs.
```

Comment:
If `ContinualTasks` really are almost identical to `MaterializedViews`, then maybe we change naming within the adapter to something like `ContinualDataflow`?